io.catenax.shared.physical_dimension/1.1.0/PhysicalDimensions.ttl

```diff
@@ -0,0 +1,104 @@
+#######################################################################
+# Copyright (c) 2023 T-Systems International GmbH
+# Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
+# Copyright (c) 2023 Fraunhofer-Institut für Produktionstechnik und Automatisierung
+#
```

Comment:
should this be added: Copyright (c) 2023 Contributors to the Eclipse Foundation?

Reply:
added